addons/analytic/models/analytic_line.py

```diff
@@ -77,6 +77,9 @@ def _get_mandatory_plans(self, company, business_domain):
     def _get_plan_domain(self, plan):
         return [('plan_id', 'child_of', plan.id)]
 
+    def _get_account_node_context(self, plan):
```

Comment:
What do you think? (should also be the case with `_get_plan_domain` just above too, IMO; but since we're not touching it, let's just leave it)

```suggestion
    @api.model
    def _get_account_node_context(self, plan):

Reply:
Actually, it's more of a nitpick. Feel free to ignore :D